packages/router/README.md

```diff
@@ -0,0 +1,37 @@
+# @talend/react-cmf-router
+
+This is the old router of react-cmf packaged as external dependency.
+
+## Setup
+
+First add it to your bootstrap app
+
+```javascript
+import cmf from '@talend/react-cmf';
+import getRouter from '@talend/react-cmf-router';
+
+const router = getRouter({ history, sagaRouterConfig });
+
+cmf.bootstrap({
+    modules: [router.cmfModule],
+    RootComponent: router.RootComponent,
+});
+```
+
+Then every cmfConnect component do not have anymore the context.router. That means you have to search for context.router in your code and update it.
```

Comment:
This has nothing to do in the module readme. This is a breaking change migration guideline

Reply:
`cmfConnect` is an higher order component, i think you should rephrase it like this:
Now components wrapped inside a `cmfConnect` higher order component don't have router's context injected as props.
This mean that you have to use the React context API yoursef in the component wrapped by `cmfConnect`

Reply:
context BR has been revert